lib/movie_wager_api/repo.ex

```diff
@@ -1,3 +1,43 @@
 defmodule MovieWagerApi.Repo do
   use Ecto.Repo, otp_app: :movie_wager_api
 end
+
+
+defmodule MovieWagerApi.Repo.Tasks do
+
+### Release Tasks:
+
+  def migrate do
+    load_app()
+    start_repo(MovieWagerApi.Repo)
+    migrations_path = Application.app_dir(:movie_wager_api, "priv/repo/migrations")
+    # Comment
+    info "Executing migrations for #{inspect MovieWagerApi.Repo} in #{migrations_path}:"
+
+    Ecto.Migrator.run(MovieWagerApi.Repo, migrations_path, :up, all: true)
+    System.halt(0)
+  end
+
+### Helper functions
+
+  defp load_app do
+    start_applications([:logger, :postgrex, :ecto])
+    :ok = Application.load(:movie_wager_api)
+  end
+
+  defp start_applications(apps) do
+    Enum.each(apps, fn app ->
+      {:ok, _} = Application.ensure_all_started(app)
```

Comment:
Same here, do we need the tuple assignment? Can it be just `Application.ensure_all_started(app)`?